hack/.ci/run-e2e-gke.sh

```diff
@@ -0,0 +1,129 @@
+    nodeSelector:
+      scylla.scylladb.com/node-type: scylla
+    tolerations:
+    - effect: NoSchedule
+      key: role
+      operator: Equal
+      value: scylla-clusters
+EOF
+
+kubectl_create -n local-csi-driver -f=https://raw.githubusercontent.com/scylladb/k8s-local-volume-provisioner/dcfceb52d9122192e50236463060cb5151d90dc4/deploy/kubernetes/local-csi-driver/{00_namespace,10_csidriver,10_driver_serviceaccount,10_provisioner_clusterrole,20_provisioner_clusterrolebinding,50_daemonset}.yaml
+kubectl -n local-csi-driver patch --field-manager="${field_manager}" daemonset/local-csi-driver --type=json -p='[{"op": "add", "path": "/spec/template/spec/nodeSelector/pool", "value":"workers"}]'
+# FIXME: The storage class should live alongside the manifests.
+kubectl_create -n local-csi-driver -f https://raw.githubusercontent.com/scylladb/k8s-local-volume-provisioner/dcfceb52d9122192e50236463060cb5151d90dc4/example/storageclass_xfs.yaml
+# TODO: Our tests shouldn't require a default storage class but rather have an option to specify which one to use
+kubectl patch --field-manager="${field_manager}" storageclass scylladb-local-xfs -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+
+SCYLLA_OPERATOR_FEATURE_GATES='AllAlpha=true,AllBeta=true'
+export SCYLLA_OPERATOR_FEATURE_GATES
+REENTRANT=true timeout -v 10m ./hack/ci-deploy.sh "${SO_IMAGE}"
+# Raise loglevel in CI.
+# TODO: Replace it with ScyllaOperatorConfig field when available.
+kubectl -n scylla-operator patch --field-manager="${field_manager}" deployment/scylla-operator --type=json -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--loglevel=4"}]'
+kubectl -n scylla-operator rollout status deployment/scylla-operator
+
+kubectl -n scylla-manager patch --field-manager="${field_manager}" deployment/scylla-manager-controller --type=json -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--loglevel=4"}]'
+kubectl -n scylla-manager rollout status deployment/scylla-manager-controller
+
+ingress_address="$( kubectl -n haproxy-ingress get svc haproxy-ingress --template='{{ .spec.clusterIP }}' )"
+
+kubectl create namespace e2e --dry-run=client -o yaml | kubectl_create -f -
+kubectl create clusterrolebinding e2e --clusterrole=cluster-admin --serviceaccount=e2e:default --dry-run=client -o yaml | kubectl_create -f -
+kubectl create -n e2e pdb my-pdb --selector='app=e2e' --min-available=1 --dry-run=client -o yaml | kubectl_create -f -
+
+kubectl -n e2e run --restart=Never --image="${SO_IMAGE}" --labels='app=e2e' --command=true e2e -- bash -euExo pipefail -O inherit_errexit -c "mkdir /tmp/artifacts && scylla-operator-tests run scylla-operator/conformance/parallel --parallelism=60 --loglevel=2 --color=false --artifacts-dir=/tmp/artifacts --feature-gates=\"${SCYLLA_OPERATOR_FEATURE_GATES}\" --override-ingress-address=\"${ingress_address}\" && touch /tmp/done && until [[ -f '/tmp/exit' ]]; do sleep 1; done"
```

Comment:
Compared to existing CI, this doesn't run serial suite at all, and there're no runs without feature flags.

Reply:
I'll add the serial suite, cost wise PRs will move to all features enabled and we can add some of the other combinations as periodics

Reply:
done

Reply:
Until we have periodics or postsubmits you've mentioned, isn't it prone to introducing regressions to only run suites with features enabled, compared to how it's set up now?

Reply:
it is. the set is fairly limited.